src/main/java/com/microsoft/sqlserver/jdbc/DataTypes.java

```diff
@@ -266,7 +269,9 @@ enum GetterConversion {
 
         SQL_VARIANT(SSType.Category.SQL_VARIANT, EnumSet.of(JDBCType.Category.CHARACTER, JDBCType.Category.SQL_VARIANT,
                 JDBCType.Category.NUMERIC, JDBCType.Category.DATE, JDBCType.Category.TIME, JDBCType.Category.BINARY,
-                JDBCType.Category.TIMESTAMP, JDBCType.Category.NCHARACTER, JDBCType.Category.GUID));
+                JDBCType.Category.TIMESTAMP, JDBCType.Category.NCHARACTER, JDBCType.Category.GUID)),
+
+        VECTOR(SSType.Category.VECTOR, EnumSet.of(JDBCType.Category.CHARACTER, JDBCType.Category.VECTOR));    
```

Comment:
(similar to above) Do we need JDBCType.Category.VECTOR in GetterConversion for the CHARACTER type, and vice versa?